main.py

```diff
@@ -53,6 +79,14 @@ async def call_api(method, url: str, body=None, params=None) -> str:
         return response.text
 
 
+def token_from_header():
+    if not BITRISE_TOKEN:
```

Comment:
Nitpick: is this check needed? Does `mcp.get_request()` throw an exception or does it return `None` if its in `stdio` mode?